arangod/RocksDBEngine/RocksDBReplicationManager.cpp

```diff
@@ -202,32 +204,32 @@ RocksDBReplicationContext* RocksDBReplicationManager::find(RocksDBReplicationId
 /// populates clientId
 //////////////////////////////////////////////////////////////////////////////
 
-int RocksDBReplicationManager::extendLifetime(RocksDBReplicationId id, 
-                                              std::string& clientId,
-                                              double ttl) {
+ResultT<std::pair<SyncerId, std::reference_wrapper<std::string const>>>
```

Comment:
Is it safe to return a reference to a string here?
That string may be deleted afterwards by other threads.

Reply:
Right, will fix that.